pkg/config/config.go

```diff
@@ -586,3 +586,94 @@ func generateOrgConfigGroup(org *Organization, mspConfig *mb.MSPConfig) (*cb.Con
 
 	return orgGroup, nil
 }
+
+// CreateSignedConfigUpdateEnvelope creates a signed configuration update envelope.
+func CreateSignedConfigUpdateEnvelope(configUpdate *cb.ConfigUpdate, signingIdentity *SigningIdentity, signatures ...*cb.ConfigSignature) (*cb.Envelope, error) {
+	if configUpdate == nil {
+		return nil, errors.New("no config update specified")
+	}
+	if signingIdentity == nil {
+		return nil, errors.New("no signing identity specified")
+	}
+	if len(signatures) == 0 {
+		return nil, errors.New("no signatures specified")
+	}
```

Comment:
I think we're getting a little validation happy. If a user gives us null references, things will panic because they gave us garbage.